### ISSUE TITLE
如何复现测试过程

### ISSUE THREAD

**Issue by 1kuzus** [lyclyc52/Instance_NeRF#3]
作者您好！我想复现您的工作中的预测部分，以得到论文中的分实例割结果，但缺少文档的指示摸索起来有些困难。请问作者可以更新一下这部分的文档嘛？或者如果不麻烦的话，也希望您能在这条issue的回复中给予我简单的指示。非常感谢！

**Comment by lyclyc52** [comment]
Thank you for your interest. To obtain the results,
1. Use the NeRF-RCNN model to get initial masks. We provide the checkpoint and the features extracted from NeRF
2. Use Mask2Former model to generate 2D mask and get the matched 2D mask. You can refer to this [README](https://github.com/lyclyc52/Instance_NeRF/tree/main/Mask2Former_sample)
3. Run Instance NeRF. You can refer toi this [README](https://github.com/zymk9/torch-ngp/tree/89ffb5e91ad00e22e3cd42de631a422198408152)

Also as suggested by this [issue](https://github.com/lyclyc52/Instance_NeRF/issues/2), our document seems hard to follow. Really sorry abut this. Thus, we plan to upload additional information later, such as Mask2Former results, NeRF-RCNN predicted results, and a clear instruction on a specific example for you to run.

**Comment by 1kuzus** [comment]
Thank you very much for the instructions and these help me a lot!

**Comment by liren2515** [comment]
Hello, is there any update on the demo?

**Comment by lyclyc52** [comment]
I updated readme file. See this [section](https://github.com/lyclyc52/Instance_NeRF/tree/main?tab=readme-ov-file#inference). Hope this can help you.

**Comment by liren2515** [comment]
> I updated readme file. See this [section](https://github.com/lyclyc52/Instance_NeRF/tree/main?tab=readme-ov-file#inference). Hope this can help you.

Hello, I have trained a Nerf model following [link](https://github.com/zymk9/torch-ngp/tree/6be6af198f1092e8d75574727a030ae15e199fe8). I want to extract features for 3D mask preparation. But I cannot find an instruction about how to do it. Could you show that to me ? Thanks!

**Comment by lyclyc52** [comment]
Feature extraction is based on instant-ngp. We do not make it consistent with our instance nerf training in this pipeline, which we use torch-ngp since it is easier to modify. Sorry about this. For feature extraction, please refer to our [NeRF-RPN](https://github.com/lyclyc52/NeRF_RPN) for details.
